UEBPrep/app/controllers/cards_controller.rb

```diff
@@ -37,6 +37,33 @@ def edit
   end
 
   # POST /cards
+  def import
+    cards_table = CSV.table((params[:file]).path)
+    cards_table.each do |row|
+      p row
+      case row[:cardtype]
+        when 'rule'
+          @content = TitledCardContent.new(:title => row[:cardtitle],
+                                           :text => row[:cardtext],
+                                           :created_by => current_user.id)
+        else #default to text card if no card type specified
```

Comment:
There is an error if no cardtype is specified, so a default is needed here. Seemed better this way than to have a redundant "when 'text'" case. Also thought about refactoring this with the create method, but one uses "row" and the other uses "params".